### ISSUE TITLE
Add an update hook to disable search_index_options module

### ISSUE THREAD

**Issue by jenlampton** [backdrop/backdrop-issues#4508]
We added the ability to prevent certain node types from being indexed in https://github.com/backdrop/backdrop-issues/issues/4197, essentially moving the search_index_options module into core.

I had several sites that were running this module when they were updated to 1.16.x, and started experiencing some serious performance issues -- especially during cron -- that caused my database server to go down.

https://backdropcms.org/project/search_index_options shows that there are still 11 sites running this module, but https://backdropcms.org/project/usage/search_index_options doesn't tell us what version of Backdrop core they are on.

Would it be worthwhile to add an update hook to core that disables this module? How has this been handled with other modules like token and ckeditor?

**Comment by klonos** [comment]
CKEditor never existed as a contrib module for Backdrop; its functionality was included in 1.0. Token was ported however, so it is included in `backdrop_merged_modules()`. See: https://github.com/backdrop/backdrop/blob/1.x/core/includes/module.inc#L987

**Comment by jenlampton** [comment]
Ckeditor wasn't added till 1.2, and there was a contrib version. I used it
:)

**Comment by klonos** [comment]
Indeed, I stand corrected: https://github.com/backdrop-contrib/ckeditor
